golos.publication/golos.publication.abi

```diff
@@ -827,6 +827,14 @@
           "orders": [
               {"field": "message_id", "order": "asc"}
           ]
+       },
+       {
+          "name": "combine",
+          "unique": "false",
```

Comment:
it's unique, it's impossible to have 2 records with the same value of (`message_id`, `voter`)